lib/src/main/java/com/cloudbees/groovy/cps/CpsTransformer.java

```diff
@@ -133,6 +135,10 @@ public void call(SourceUnit source, GeneratorContext context, ClassNode classNod
                 visitNontransformedStatement(statement);
             }
 
+            if (!classNode.declaresInterface(SERIALIZABLE_TYPE)) {
```

Comment:
Well, it could be inheriting `Serializable` from a supertype; did you test that? For that matter, does it hurt anything to unconditionally `addInterface(SERIALIZABLE_TYPE)`?

Reply:
Yup, you're right, that'll work.